circle.yml

```diff
@@ -0,0 +1,19 @@
+checkout:
+  post:
```

Comment:
It might be worth leaving a little comment here explaining why this is necessary.